pkgs/tools/admin/ansible/default.nix

```diff
@@ -0,0 +1,59 @@
+{ stdenv, lib, fetchurl, python2Packages }:
+
+let
+  generic = { version, sha256 }:
+  python2Packages.buildPythonApplication rec {
+    name = "ansible-${version}";
+
+    src = fetchurl {
+      url = "http://releases.ansible.com/ansible/${name}.tar.gz";
+      inherit sha256;
+    };
+
+    prePatch = ''
+      sed -i "s,/usr/,$out," lib/ansible/constants.py
+    '';
+
+    doCheck = false;
+    dontStrip = true;
+    dontPatchELF = true;
+    dontPatchShebangs = false;
+    windowsSupport = true;
+
+    propagatedBuildInputs = with python2Packages; [
+      pycrypto paramiko jinja2 pyyaml httplib2 boto six
+      netaddr dns
+    ] ++ lib.optional windowsSupport pywinrm;
+
+    meta = with stdenv.lib; {
+      homepage = "http://www.ansible.com";
+      description = "A simple automation tool";
+      license = with licenses; [ gpl3] ;
+      maintainers = with maintainers; [
+        jgeerds
+        joamaki
+      ];
+      platforms = with platforms; linux ++ darwin;
+    };
+  };
+
+in rec {
+
+  ansible1 = generic {
+    version = "1.9.6";
+    sha256  = "0pgfh5z4w44sjgd77q6k769a5ipigjlm28zbpf2jhvz7n60kfxsh";
+  };
+
+  ansible2_1 = generic {
+    version = "2.1.4.0";
+    sha256  = "05nc68900qrzqp88970j2lmyvclgrjki66xavcpzyzamaqrh7wg9";
+  };
+
+  ansible2_2 = generic {
+    version = "2.2.1.0";
+    sha256  = "0gz9i30pdmkchi936ijy873k8di6fmf3v5rv551hxyf0hjkjx8b3";
+  };
+
+  ansible  = ansible1;
```

Comment:
I don't know of anything official from RedHat, but they specifically mentioned only releasing fixes for the 2.1 and 2.2 versions hence I'm assuming those are the only properly supported versions.

As for changing the default - that would only be in unstable, correct?